lib/src/git.rs

```diff
@@ -894,24 +895,30 @@ impl<'a> RemoteCallbacks<'a> {
                     // Try to get the SSH key from the agent once. We don't even check if
                     // $SSH_AUTH_SOCK is set because Windows uses another mechanism.
                     if !tried_ssh_agent {
-                        tracing::info!(username, "using ssh_key_from_agent");
```

Comment:
I also changed the message to say "trying ssh_key <key>" and "trying ssh_key_from_agent"